src/CMakeLists.txt

```diff
@@ -39,6 +39,10 @@
 # POSSIBILITY OF SUCH DAMAGE.
 # 
 ###############################################################################
+if(RAJA_ENABLE_AGENCY)
+  set(AGENCY_DIR "${PROJECT_SOURCE_DIR}/extra/agency")
+  include_directories(${AGENCY_DIR})
```

Comment:
It seems like we need to do this in every CMake file, as `include_directories` isn't inherited with nested CMakeLists.txt files